### PR TITLE
Improve social media icons [MAILPOET-5582]

### PR DESCRIPTION
**Pull request by pavel-mailpoet**
## Description

Remove the outdated icons and add a new Twitter name and icon.

## Code review notes

_N/A_

## QA notes

_N/A_

## Linked PRs

_N/A_

## Linked tickets

[MAILPOET-5582]

## After-merge notes

_N/A_

## Tasks

- [ ] I followed [best practices](https://codex.wordpress.org/I18n_for_WordPress_Developers) for translations
- [ ] I added sufficient test coverage
- [ ] I embraced TypeScript by either creating new files in TypeScript or converting existing JavaScript files when making changes

[MAILPOET-5582]: https://mailpoet.atlassian.net/browse/MAILPOET-5582?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: mailpoet/assets/img/newsletter_editor/social-icons/03-circles/Twitter.png
Comment (lysyjan) [inline mailpoet/assets/img/newsletter_editor/social-icons/03-circles/Twitter.png:1]: The new file seems to be bigger than needed. It is much bigger compared with the previous one. The old one was 64 x 64px, and the new one is 512 x 512px. Should we resize it? What do you think?

**Review by lysyjan — COMMENTED** [review]
@pavel-mailpoet Good job. I have only one questions. Please, see the comment

**Review by lysyjan — APPROVED** [review]
@pavel-mailpoet Thanks. LGTM! I'm approving, but the trunk branch contains a failing test, so a rebase will be needed. Nevertheless, QA can test it in the meantime.

**Comment by Aschepikov** [comment]
Hi @pavel-mailpoet there is one flaky failing test, can you rebase it pls maybe it will help

**Comment by lysyjan** [comment]
Hi @Aschepikov! Because Pavel is AFK this week, I rebased it.

**Comment by Aschepikov** [comment]
@lysyjan great, thank you

**Review by Aschepikov — APPROVED** [review]

**Pull request merged**